### PR TITLE
Update SQL query examples for schema 40

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Update SQL query examples for schema 40

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/89557 https://github.com/home-assistant/core/pull/89676
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/sql.markdown [hunk 1]
```diff
@@ -126,7 +157,31 @@ Use `state` as column for value.
...
+        SELECT
+          metadata_id
+        FROM
+          states_meta
+        WHERE
+          entity_id = 'sensor.temperature_in'
+      )
+      AND old_state_id IS NOT NULL
+    ORDER BY
+      state_id DESC
```
Comment (bdraco) [inline source/_integrations/sql.markdown]:
```suggestion
      last_updated_ts DESC
```

File: source/_integrations/sql.markdown [hunk 2]
```diff
@@ -126,7 +157,31 @@ Use `state` as column for value.
...
+FROM
+  states
+WHERE
+  state_id == (
+    SELECT
+      states.old_state_id
+    FROM
+      states
+    WHERE
+      metadata_id == (
```
Comment (bdraco) [inline source/_integrations/sql.markdown]:
```suggestion
      metadata_id = (
```

File: source/_integrations/sql.markdown [hunk 3]
```diff
@@ -126,7 +157,31 @@ Use `state` as column for value.
 
 Based on previous example with temperature, the query to get the former state is :
 ```sql
-SELECT * FROM (SELECT * FROM states WHERE entity_id = 'sensor.temperature_in' ORDER BY state_id DESC LIMIT 2) two_entity ORDER BY state_id ASC LIMIT 1;
+SELECT
+  states.state
+FROM
+  states
+WHERE
+  state_id == (
```
Comment (bdraco) [inline source/_integrations/sql.markdown]:
```suggestion
  state_id = (
```

File: source/_integrations/sql.markdown [hunk 4]
```diff
@@ -117,7 +132,23 @@ sensor:
 The query will look like this:
 
 ```sql
-SELECT * FROM states WHERE entity_id = 'sensor.temperature_in' ORDER BY state_id DESC LIMIT 1;
+SELECT
+  states.state
+FROM
+  states
+WHERE
+  metadata_id == (
```
Comment (bdraco) [inline source/_integrations/sql.markdown]:
```suggestion
  metadata_id = (
```

File: source/_integrations/sql.markdown [hunk 5]
```diff
@@ -35,11 +35,26 @@ To enable it, add the following lines to your `configuration.yaml` file (example
...
-      DESC LIMIT 1;
+      SELECT
+        states.state
+      FROM
+        states
+        LEFT JOIN state_attributes ON (
+          states.attributes_id = state_attributes.attributes_id
+        )
+      WHERE
+        metadata_id == (
```
Comment (bdraco) [inline source/_integrations/sql.markdown]:
```suggestion
        metadata_id = (
```

**Comment by bdraco** [comment]
We need to add an index for `entity_id` and `event_type` even though we aren't likely to need them since we have the LRU.

I'll adjust that before moving this forward

**Review by frenck — APPROVED** [review]

**Comment by bdraco** [comment]
thanks

**Pull request merged**